src/main/java/mate/academy/spring/validation/EmailValidator.java

```diff
@@ -1,17 +1,19 @@
 package mate.academy.spring.validation;
 
+import java.util.regex.Pattern;
 import javax.validation.ConstraintValidator;
 import javax.validation.ConstraintValidatorContext;
 import mate.academy.spring.validation.annotation.Email;
 
 public class EmailValidator implements ConstraintValidator<Email, String> {
-    private static final String EMAIL_PATTERN = "^(?=.{1,64}@)[A-Za-z0-9_-]"
+    private static final Pattern EMAIL_PATTERN = Pattern
+            .compile("^(?=.{1,64}@)[A-Za-z0-9_-]"
             + "+(\\.[A-Za-z0-9_-]+)*@[^-][A-Za-z0-9-]"
-            + "+(\\.[A-Za-z0-9-]+)*(\\.[A-Za-z]{2,})$";
+            + "+(\\.[A-Za-z0-9-]+)*(\\.[A-Za-z]{2,})$");
 
     @Override
     public boolean isValid(String email,
                            ConstraintValidatorContext constraintValidatorContext) {
-        return email != null && email.matches(EMAIL_PATTERN);
+        return email != null && email.matches(String.valueOf(EMAIL_PATTERN));
```

Comment:
What did you just do?)) You have created an object of a type Pattern, from which created a String object to finally again create a Pattern object inside the matches method. Use next construction `EMAIL_PATTERN.matcher(email).matches()`